requirements-dev.txt

```
@@ -18,3 +18,4 @@ codecov
 protobuf
 nbval
 py3Dmol
+garnett
```

I strongly recommend using the latest version 0.7.1, which has some API breakage from 0.6. Also 0.7.0 was a broken release (its dependencies were specified incorrectly).
```suggestion
garnett >=0.7.1
```